src/test/ui/issues/issue-61108.stderr

```diff
@@ -6,11 +6,17 @@ LL |     let mut bad_letters = vec!['e', 't', 'o', 'i'];
 LL |     for l in bad_letters {
    |              -----------
    |              |
-   |              value moved here
+   |              value moved due to this method call
```

Comment:
It would be interesting to see if we could detect the `DesugaringKind` in order to customize these messages for something like "value moved here due to an implicit call to `into_iter`".